base/server/tomcat-8.5/conf/server.xml

```diff
@@ -171,9 +171,6 @@ Tomcat Port         = [TOMCAT_SERVER_PORT] (for shutdown)
                ocspMaxCacheEntryDuration="14400"
                ocspTimeout="10"
                strictCiphers="true"
-               sslVersionRangeStream="[TOMCAT_SSL_VERSION_RANGE_STREAM]"
-               sslVersionRangeDatagram="[TOMCAT_SSL_VERSION_RANGE_DATAGRAM]"
-               sslRangeCiphers="[TOMCAT_SSL_RANGE_CIPHERS]"
```

Comment:
Should `strictCiphers` be removed too?

Do we need an upgrade script to remove these parameters from existing instances?

Reply:
Strict ciphers will be removed as part of Tomcat minification, so yes, this should go too.

Reply:
And yes, probably need an upgrade script, I'll look into that.